topics/Maven.xml

```diff
@@ -314,6 +314,11 @@
     </dependencies>
 </project>
                 ]]></code>
+                <tip>
+                    <p>
+                        To load changes made in <path>pom.xml</path>, you can click <control>Load Maven Changes</control> at the right part of the editor.
```

Comment:
To load changes made -> To have the changes made take effect, you can click the ....., on the Maven Tool Window.

Reply:
It's better to describe what the effect is.

Reply:
rephrased a bit and added a link to IDEA docs